python/src/cloudstorage/storage_api.py

```diff
@@ -169,6 +169,43 @@ def get_bucket_async(self, path, **kwds):
     """GET a bucket."""
     return self.do_request_async(self.api_url + path, 'GET', **kwds)
 
+  # pylint: disable=too-many-locals
+  def compose_object(self, file_list, destination_file, content_type):
+    """COMPOSE multiple objects together.
+
+    Taking a list of files it calls the put object with the compose flag
```

Comment:
Alternative: Using the given list of files calls the put object with the compose flag.

and decide about a comma or a dot and Uppercase bellow.